# state of ackerman linkage tranforms

# from Ground to Left-Steering-Arm [rad]
Angle theta_2

# from Rack to Left-Tie-Bar [rad]
Angle theta_3

# from Ground to Right-Steering -rm [rad]
Angle theta_4

# from Rack to Right-Tie-Bar [rad]
Angle theta_5

# rotation of Pinion [rad]
Angle theta_pin

# displacement of rack [m]
float64 d
